### PR TITLE
Add LSTM model.

### PR DESCRIPTION
**Pull request by gtoderici**
Adding a baseline LSTM model.

### CODE REVIEW

File: frame_level_models.py [hunk 1]
```diff
@@ -191,3 +193,48 @@ def create_model(self,
...
         vocab_size=vocab_size,
         **unused_params)
+
+class FrameLevelLSTMLogisticModel(models.BaseModel):
+
+  def create_model(self, model_input, vocab_size, num_frames, **unused_params):
+    """Creates a model which uses a logistic classifier over the output
+    of stacked LSTM layers.
+
+    This class is intended to be an example for implementors of frame level
```
Comment (LeegleechN) [inline frame_level_models.py]: remove this paragraph

File: frame_level_models.py [hunk 2]
```diff
@@ -191,3 +193,48 @@ def create_model(self,
...
+            state_is_tuple=False)
+
+    loss = 0.0
+    with tf.variable_scope("RNN"):
+      outputs, state = tf.nn.dynamic_rnn(stacked_lstm, model_input,
+                                         sequence_length=num_frames,
+                                         dtype=tf.float32)
+    output = slim.fully_connected(
+        state, vocab_size, activation_fn=tf.nn.sigmoid,
+        weights_regularizer=slim.l2_regularizer(0.01), scope='final_softmax')
```
Comment (LeegleechN) [inline frame_level_models.py]: you named it a softmax but it's not a softmax.

File: frame_level_models.py [hunk 3]
```diff
@@ -191,3 +193,48 @@ def create_model(self,
...
+                for _ in range(number_of_layers)
+                ],
+            state_is_tuple=False)
+
+    loss = 0.0
+    with tf.variable_scope("RNN"):
+      outputs, state = tf.nn.dynamic_rnn(stacked_lstm, model_input,
+                                         sequence_length=num_frames,
+                                         dtype=tf.float32)
+    output = slim.fully_connected(
```
Comment (LeegleechN) [inline frame_level_models.py]: Can you copy the section from the end of the DBoF implementation which uses the "video_level_classifier_model" flag?

File: frame_level_models.py (hunk 1 above)
Comment (gtoderici) [inline frame_level_models.py]: Done.

File: frame_level_models.py (hunk 2 above)
Comment (gtoderici) [inline frame_level_models.py]: Good point :)

File: frame_level_models.py (hunk 3 above)
Comment (gtoderici) [inline frame_level_models.py]: Cool. This will make it more flexible.

File: frame_level_models.py [hunk 4]
```diff
@@ -191,3 +193,48 @@ def create_model(self,
         model_input=activation,
         vocab_size=vocab_size,
         **unused_params)
+
+class FrameLevelLSTMLogisticModel(models.BaseModel):
+
+  def create_model(self, model_input, vocab_size, num_frames, **unused_params):
+    """Creates a model which uses a logistic classifier over the output
```
Comment (LeegleechN) [inline frame_level_models.py]: Remove the specification of "logistic classifier" since it will be controllable through flags.
